community/kernel/src/main/java/org/neo4j/kernel/impl/store/NeoStores.java

```diff
@@ -694,4 +440,132 @@ public void assertOpen()
+                (DynamicArrayStore) getOrCreateStore(StoreType.NODE_LABEL) ) );
+    }
+
+    CommonAbstractStore createPropertyKeyTokenStore( String storeName )
+    {
+        File storeFile = getStoreFile( storeName );
+        return initialize( new PropertyKeyTokenStore( storeFile, config, idGeneratorFactory,
+                pageCache, logProvider, (DynamicStringStore) getOrCreateStore( StoreType.PROPERTY_KEY_TOKEN_NAME )
+        ) );
+    }
+
+    CommonAbstractStore createPropertyStore( String storeName )
+    {
+        File storeFile = getStoreFile( storeName );
+        return initialize( new PropertyStore( storeFile, config, idGeneratorFactory, pageCache, logProvider,
+                (DynamicStringStore) getOrCreateStore( StoreType.PROPERTY_STRING ),
+                (PropertyKeyTokenStore) getOrCreateStore( StoreType.PROPERTY_KEY_TOKEN ),
+                (DynamicArrayStore) getOrCreateStore( StoreType.PROPERTY_ARRAY ) ) );
+    }
+
+    CommonAbstractStore createRelationshipStore( String storeName )
+    {
+        File file = getStoreFile( storeName );
+        return initialize( new RelationshipStore( file, config, idGeneratorFactory, pageCache, logProvider ) );
+    }
+
+    CommonAbstractStore createDynamicStringStore( String storeName, IdType idType,
+            Setting<Integer> blockSizeProperty )
+    {
+        return createDynamicStringStore( storeName, idType, config.get( blockSizeProperty ) );
+    }
+
+    CommonAbstractStore createDynamicStringStore( String storeName, IdType idType, int blockSize )
+    {
+        File storeFile = getStoreFile( storeName );
+        return initialize( new DynamicStringStore( storeFile, config, idType, idGeneratorFactory,
+                pageCache, logProvider, blockSize ) );
+    }
+
+    CommonAbstractStore createRelationshipTypeTokenStore( String storeName )
+    {
+        File storeFile = getStoreFile( storeName );
+        return initialize( new RelationshipTypeTokenStore( storeFile, config, idGeneratorFactory,
+                pageCache, logProvider, (DynamicStringStore) getOrCreateStore( StoreType.RELATIONSHIP_TYPE_TOKEN_NAME
+        ) ) );
+    }
+
+    CommonAbstractStore createLabelTokenStore( String storeName )
+    {
+        File fileName = getStoreFile( storeName );
+        return initialize( new LabelTokenStore( fileName, config, idGeneratorFactory, pageCache,
+                logProvider, (DynamicStringStore) getOrCreateStore( StoreType.LABEL_TOKEN_NAME ) ) );
+    }
+
+    CommonAbstractStore createSchemaStore( String storeName )
+    {
+        File fileName = getStoreFile( storeName );
+        return initialize( new SchemaStore( fileName, config, IdType.SCHEMA, idGeneratorFactory, pageCache,
+                logProvider ) );
+    }
+
+    CommonAbstractStore createMetadataStore()
+    {
+        return initialize( new MetaDataStore( neoStoreFileName, config, idGeneratorFactory, pageCache, logProvider ) );
+    }
```

Comment:
I'd move `createMetaDataStore` bellow `createCountsStore` so the order roughly matches the enum.